meeteval/der/md_eval.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import re
+import decimal
+import shutil
+import tempfile
+import subprocess
+import dataclasses
+from pathlib import Path
+
+import meeteval.io
+from meeteval.wer.wer.error_rate import ErrorRate
+
+
+def _fix_channel(r):
+    return meeteval.io.rttm.RTTM([
+        line.replace(channel='1')  # Thilo puts there usually some random value, e.g. <NA> for hyp and 0 for ref, while common default is 1
```

Comment:
Can you change the default in the `RTTM` class? 

I think the comment should say that the original (?) RTTM definition (Appendix A in "The 2009 (RT-09) Rich Transcription Meeting Recognition Evaluation Plan") doesn't say anything about the channel format or defaults, but dscore enforces a "1" (https://github.com/nryant/dscore#rttm)

Reply:
Done and added your comment.